### ISSUE TITLE
WSG - Battleground

### ISSUE THREAD

**Issue by serverwar** [liyunfan1223/mod-playerbots#459]
![image](https://github.com/user-attachments/assets/ca6ab096-27f9-4a7f-ade2-91b1fbdba5ec) Bot players are gathering at the flag base, with 5-6 of them there.

**Comment by fuzzdeveloper** [comment]
Are the bots gathering in enemy base or their own? I'm guessing its happening when both flags are taken (judging from your addon, which I'm not familiar with, but I'm guessing that's what its saying)

**Comment by serverwar** [comment]
> Are the bots gathering in enemy base or their own? I'm guessing its happening when both flags are taken (judging from your addon, which I'm not familiar with, but I'm guessing that's what its saying)

Yes, at the enemy base, but they become too much in one place, they don't identify which flag was taken and they need to recover it.

**Comment by EricksOliveira** [comment]
Yes. Their focus is the enemy flag. So much so that they become full enemy base. The ideal would be to have more active movement.

**Comment by Dreathean** [comment]
@serverwar Can you confirm if this is still an issue for you?
